compose.yml

```diff
@@ -0,0 +1,24 @@
+name: SyllabusX
+services:
+  client:
+    build:
+      context: ./
+      dockerfile: Dockerfile  
+    ports:
+      - 3000:3000
+    environment:
```

Comment:
remove these as .env is being copied into container so next will be using those env variables, because of this my .env was not being recognized

Reply:
ok @akshat-OwO